src/python/WMComponent/TaskArchiver/TaskArchiverPoller.py

```diff
@@ -214,8 +214,7 @@ def completeTasks(self, finishedwfs):
 
         centralCouchAlive = True
         try:
-            abortedWorkflows = self.killCondorJobsByWFStatus(["aborted"])
-            self.killCondorJobsByWFStatus(["force-complete"])
+            self.killCondorJobsByWFStatus(["force-complete", "aborted"])
```

Comment:
Shouldn't we add aborted-completed too? There is still the possibility of having jobs for a workflow in completed though.
BTW, this will kill any job types for those workflows, right? In theory we should let LogCollect and Cleanup jobs running.

Reply:
we only checked aborted since after this aborted  workflow supposed to be change to aborted-completed.
As far as I know we don't kill log collect and cleanup jobs.

Reply:
And the reason we kill on force-completed and aborted only - at this stage - is because `killWMBSWorkflow` from WorkQueueManager will kill subscriptions and also jobs from condor. Is that correct or I'm missing something?

Reply:
Yes, that is right. Then it will move to aborted-completed, completed state after that. 
In principle, it doesn't make sense to check aborted-completed, completed at this point.